packages/generator-single-spa/src/react/templates/.babelrc

```diff
@@ -4,5 +4,9 @@
     "@babel/preset-react"
   ],
   "plugins": [
+    ["@babel/plugin-transform-runtime", {
+      "useESModules": true,
```

Comment:
This seems to create some bundle savings for webpack bundles - https://babeljs.io/docs/en/next/babel-plugin-transform-runtime.html#useesmodules